### PR TITLE
starting UI component for more granular simulation logs

### PR DESCRIPTION
**Pull request by jonrkarr**
Started UI components for simulation logs

Still todo
- [x] Sort SED documents/tasks/outputs by their ids
- [x] Add internal links to table of contents
- [x] Display executed algorithm for each task
- [x] Display simulator details for each task
- [x] Black background, white text for stdout/err logs
- [x] Add summary to left column
- [x] Use accordion folding to better organize content
- [x] Add `.spec` files for new components
- [x] Copy similar styling to simulators test suite cases
    - [x] Heading colors for skipped, failed
    - [x] Accordion folding

### CODE REVIEW

File: biosimulations/libs/dispatch/api-models/src/lib/simulationRunLog.ts [hunk 1]
```diff
@@ -111,6 +111,12 @@ export class SedTaskLog {
 
   @ApiProperty({ type: Number, example: 10.5, nullable: true })
   duration!: number | null;
+
+  @ApiProperty({ type: String, example: 'KISAO_0000019', nullable: true })
+  algorithm: string | null;
+  
+  @ApiProperty({ type: Object, example: {method: 'simulator.cvode', arguments: {relTol: 1e-6, absTol: 1e-8}}, nullable: true })  
+  simulatorDetails: {[key: string]: any} | nullable;
```
Comment (bilalshaikh42) [inline biosimulations/libs/dispatch/api-models/src/lib/simulationRunLog.ts]: i believe this should be null, not nullable

**Review by bilalshaikh42 — CHANGES_REQUESTED** [review]

File: biosimulations/libs/dispatch/api-models/src/lib/simulationRunLog.ts (hunk 1 above)
Comment (jonrkarr) [reply biosimulations/libs/dispatch/api-models/src/lib/simulationRunLog.ts]: Correct.

Comment (jonrkarr) [reply biosimulations/libs/dispatch/api-models/src/lib/simulationRunLog.ts]: Fixed

**Comment by jonrkarr** [comment]
Almost done. Just want to do two last things
- Use an accordion to fold up the details.
- Copy stylistic changes to test cases for simulation tools
  - Heading colors for skipped, failed
  - Folding

**Comment by jonrkarr** [comment]
I think this is ready to merge. I don't intend to make additional changes at this point.

There's just one small issue I couldn't get right. The logs for each individual task/report/plot are folded up. There's an icon button to fold/unfold each item. The icon should toggle between and up and down arrow. There's an issue with change detection, but I couldn't figure out how to correct it.

**Pull request merged**